src/components/output/custom-els/MultiPanel/styles.css

```diff
@@ -0,0 +1,11 @@
+multi-panel > .panel-heading {
+  background:gray;
+}
+multi-panel > .panel-content {
+  height:0px;
+  overflow:scroll;
+  transition: height 1s;
+}
+multi-panel > .panel-content[expanded] {
+  height:auto;
```

Comment:
I like this approach because it is only attribute toggle & not editing CSS from JavaScript.

However, this means user of this component should only set height of contents _inside_ the `.panel-content` element and not  `.panel-content` itself. 

thoughts ?

```html
<div>header</div>
<div class="panel-content">
  <div id='container'>set height however you like</div>
</div>
```